lua/entities/gmod_tardis/modules/sh_health.lua

```diff
@@ -241,6 +241,10 @@ if SERVER then
     end)
 
     ENT:AddHook("PhysicsCollide", "Health", function(self, data, collider)
+        if self:IsPlayerHolding() then
+            self:SetVelocity(Vector(0,0,0))
+            self:ForcePlayerDrop()
```

Comment:
I'm not sure if dropping it is a good idea, perhaps we should just ignore collisions above a really high speed when the player is holding the TARDIS? I still think you should be able to move it around with the physgun we just want to avoid the physics engine glitching out and registering extremely high speed hits by a player forcing it into the ground / wall / etc